tests/test_parse_affected.py

```diff
@@ -52,7 +52,11 @@ def main():
 def parse_affected(cur_dir, env_yaml):
     for rule_dir in ssg.rules.find_rule_dirs(cur_dir):
         rule_path = os.path.join(rule_dir, "rule.yml")
-        rule = ssg.build_yaml.Rule.from_yaml(rule_path, env_yaml)
+        try:
+            rule = ssg.build_yaml.Rule.from_yaml(rule_path, env_yaml)
+        except ssg.build_yaml.DocumentationNotComplete:
+            # Happens on non-debug build when a rule is "documentation-incomplete"
+            continue
```

Comment:
@evgenyz @matejak There were conflicts from #4945 and #4962. 
Please double check that I resolved it correctly.

Reply:
it looks like it is correct, I added this piece of code actually.